examples/label_example/search_utils.py

```diff
@@ -0,0 +1,57 @@
+'''
+this file defines search functions for searching data in elasticsearch.
```

Comment:
Any specific reason why we aren't using `LastUtteranceSearcher` processor to search through elastic search and return the links for documents?

Reply:
Here is what I think:

We create a search engine by streamlit, and I think it is a simple and easy way to wrap the search into method and when the user clicks "Query", this method is called directly to query and then displayed the results on the interface

Reply:
I feel its just redundant if we already have this functionality implemented in our processors and we still write another method to achieve the same thing, but it could be simpler and hence could be used for just the demo. You should confirm this with hector once. @hunterhector